examples/hello-world-pyprojecttoml/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["setuptools", "wheel", "setuptools-rust"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "hello-world"
+version = "1.0"
+
+[project.scripts]  # Python entry-point wrapper to be installed in `$venv/bin`
+sum-cli = "hello_world.sum_cli:main"
```

Comment:
It might be a nice example to move this to be a Rust `#[pyfunction]` as the entry point, similar to https://www.maturin.rs/bindings#both-binary-and-library

Reply:
Thank you very much @davidhewitt, I can address this in a follow up commit that will also try to tackle the CI linter errors.

Would it be OK to add if I add a second entry-point. I think it is nice to show that you can mix and match Python and Rust modules...

Reply:
Sure, having both a Python entry-point and a Rust entry point suits me 👍

Reply:
Done in https://github.com/PyO3/setuptools-rust/pull/348/commits/103993f2370b5cf25725a3a8f603bc8476509cf4